critiquebrainz/frontend/forms/review.py

```diff
@@ -19,14 +19,29 @@ def __call__(self, form, field):
         if l < self.min or self.max != -1 and l > self.max:
             raise ValidationError(self.message)
 
+# Loading Frequently Used Languages
+frequently_used_languages_raw = ['ar','zh','cs','da','nl','en','fi','fr','de','el','it','ja','ko','pl','pt','ru','es','sv','tr']
```

Comment:
I think that rather than defining a separate list here, it would be good to get the frequently used languages from the database of the reviews (using a SQL query for fetching the count of the languages used in reviews in descending order: see https://stackoverflow.com/questions/1402682/how-to-sort-by-count-with-postgresql and https://www.postgresql.org/docs/8.2/static/functions-aggregate.html). Since, the no. of reviews presently is not much, I think we can go for this function. I'd suggest to try writing a function in the `db/review.py` file for getting most used languages and using it here.